docs/install-with-docker.md

```diff
@@ -3,35 +3,21 @@ tableofcontents: 1
 redirect_from: "/Install-with-Docker"
 ---
 
-# Install with Docker
+# Build with Docker
 
 Welcome to the AzerothCore Docker guide!
 
-## Introduction
-
-Installing AzerothCore using Docker is a simplified procedure that has several benefits:
-
-- It's very easy! Docker will do all the dirty work for you.
-- It can be done in all operating systems where Docker is available (including **Windows**, **GNU/Linux**, **macOS**)
-- You don't need to install many dependencies (forget about _visual studio_, _cmake_, _mysql_, etc.. they are **NOT** required)
-- Forget about platform-specific bugs. When using Docker, AzerothCore will always run in **Linux-mode**. 
-- There are many other [benefits when using Docker](https://www.google.com/search?q=docker+benefits)
```

Comment:
what was wrong with this?

Reply:
I felt it was fluff and that this document should be describing how to deploy AzerothCore with Docker and that describing the benefits of it in this way was out of scope of the document. 

I'm not married to removing this section, I could just clean it up

1. Remove this bullet point. It doesn't actually tell the reader anything
2. Noting that it's cross platform/azerothcore in docker is supported (or at least, I've been supporting it for these platforms :D) cross platform is nice. Probably reword to clarify this point. 
3. remove parentheses and maybe clarify the original statement
4. if it's always running in "linux-mode", wouldn't platform specific-bugs for linux happen?
5. remove this bullet point. the reader can google "docker benefits" on their own